app/plugin_api_test.go

```diff
@@ -556,6 +556,36 @@ func TestPluginAPIUserCustomStatus(t *testing.T) {
 	userCs, err = th.App.GetCustomStatus(user1.Id)
 	assert.Nil(t, err)
 	assert.Equal(t, csClear, userCs)
+
+	// Set a custom status for clearing it later
+	custom.Emoji = ":100"
```

Comment:
```suggestion
	custom.Emoji = ":100:"
```

Reply:
Should we have some validation on the custom status update logic to make sure we're using a valid emoji?

Reply:
Actually my bad, we dont need the colons at all, 
https://github.com/mattermost/mattermost-webapp/blob/736fab13a828e1114e4fb1fc0909eb2f773297fe/packages/mattermost-redux/src/selectors/entities/emojis.ts#L43